packages/amplify-graphql-conversation-transformer/src/resolvers/templates/assistant-messages-subscription-resolver-fn.template.js

```diff
@@ -32,7 +32,7 @@ export function request(ctx) {
   if (!isAuthorized && ctx.stash.authFilter.length === 0) {
     util.unauthorized();
   }
-  ctx.args.filter = { ...ctx.args.filter, and: [{ conversationId: { eq: ctx.args.conversationId } }] };
+  ctx.args.filter = { ...ctx.stash.authFilter, and: [{ conversationId: { eq: ctx.args.conversationId } }] };
```

Comment:
Looks we are overwriting filter on the arguments with the computed auth filter. Is that intentional?